spec.bs

```diff
@@ -120,7 +120,7 @@ typedef (USVString or URLPatternInit) URLPatternInput;
 
 [Exposed=(Window,Worker)]
 interface URLPattern {
-  constructor(optional URLPatternInput input = {}, optional USVString baseURL);
+  constructor(optional URLPatternInput input = {}, optional USVString baseURL, optional URLPatternOptions options = {});
```

Comment:
I think we also need a second constructor form to do what you want:

```
 constructor(optional URLPatternInput input = {}, optional URLPatternOptions options = {});
```

Otherwise you cannot specify options without also specifying a base URL second arg.

Reply:
I'm going to fix this in a follow-up commit.